src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -231,17 +256,15 @@ def create_lv(name, group, size=None, **tags):
         ])
 
     lv = get_lv(lv_name=name, vg_name=group)
-    ceph_tags = {}
-    for k, v in tags.items():
-        ceph_tags['ceph.%s' % k] = v
-    lv.set_tags(ceph_tags)
```

Comment:
Are we removing the ``ceph.`` prefix for all our tags now?

Reply:
no, I am just enforcing consumers of this to pass tags as a dictionary with keys that match (I explain this a bit in the commit). Before the signature was `**tags` but that allows python-safe keys which made `block.wal` a no-go. In order to change this, `tags` should be a dict now, and there is no "helper" to translate "key" into "ceph.key"

This puts the burden of ensuring `ceph.key-name` onto the consumer.